software/firmware/version.py

```diff
@@ -0,0 +1,2 @@
+# EuroPi firmware version
+__version__ = "0.6.1"
```

Comment:
[discussion] I'm just noticing that there are two version files. Right now we release both the core and contrib together with the same process and version number. Can we combine these files? It would be nice to have only one place to changes as well as reinforce the fact that they are in sync.

Reply:
Yeah, I agree. The original idea was since there are two separate distributables then there should be separate version numbers, but in practice we have always bumped and released them together. I'm in favor of combining these for simplicity sake. If a practical need for separate version numbers arises, we can reevaluate at that time.